backend/EasyMeets.Core/EasyMeets.Core.Common/DTO/Availability/NewAvailability/NewAdvancedSlotSettingsDto.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace EasyMeets.Core.Common.DTO.Availability;
+
+public class NewAdvancedSlotSettingsDto
+{
+    public int ActivityType { get; set; }
+    public int Days { get; set; }
+    public DateTimeOffset StartDate { get; set; }
+    public DateTimeOffset EndDate { get; set; }
+    public int MaxNumberOfBookings { get; set; }
+    public int PaddingBeforeMeeting { get; set; }
+    public int BookingScheduleBlockingTimeMeetingInHours { get; set; }
+    public int Color { get; set; }
```

Comment:
Why don't  you use enum here?

Reply:
Can we reference Dal project in common?

Reply:
Enum must be in some shared module

Reply:
So should I move enums to Common project and reference it in Dal?

Reply:
Yep